pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -4,18 +4,18 @@ description: >-
   Testnet Locally
 ---
 
-# Launching your parachain manually
+# Launch a Chain on a Local Test Network
 
 ## Introduction
 
 [Paseo](https://x.com/PaseoNetwork) is the community-run Polkadot Relay chain Testnet.
 
-A typical development workflow for launching your parachain on Polkadot:&#x20;
+A typical development workflow for launching your chain on Polkadot:&#x20;
 
-* Run your parachain **locally** using Pop CLI.&#x20;
-  * Under the hood, Pop CLI launches your parachain to Paseo Local Testnet automatically for development purposes:
+* Run your chain **locally** using Pop CLI.&#x20;
+  * Under the hood, Pop CLI launches your chain to Paseo Local Testnet automatically for development purposes:
     * [Run your parachain on Paseo](running-your-parachain.md)
-* When ready to test your parachain in a live environment with other parachains:
+* When ready to test your chain in a live environment with other chains:
   * Use this guide to mimic the manual onboarding process for Paseo Testnet locally.&#x20;
   * When comfortable with manually onboarding locally, then use the next guide to onboard to Paseo Live Testnet.
     * [Launching your parachain on Paseo Testnet](launching-your-parachain-on-polkadot/launching-on-paseo.md)
```

Comment:
And then here it is called `Paseo Testnet`, a bit confusing to me whether this is Live Testnet or Local Testnet.